src/test/test.js

```diff
@@ -83,39 +86,38 @@ describe('watsonwork-echo', () => {
         return;
       }
     };
-
     // Create the Echo Web app
-    echo.webapp('testappid', 'testsecret', 'testwsecret', (err, app) => {
-      expect(err).to.equal(null);
-
-      // Listen on an ephemeral port
-      const server = app.listen(0);
-
-      // Post a Webhook challenge request to the app
-      post('http://localhost:' + server.address().port + '/echo', {
-        headers: {
-          // Signature of the test body with the Webhook secret
-          'X-OUTBOUND-TOKEN':
-            'f51ff5c91e99c63b6fde9e396bb6ea3023727f74f1853f29ab571cfdaaba4c03'
-        },
-        json: true,
-        body: {
-          type: 'verification',
-          challenge: 'testchallenge'
-        }
-      }, (err, res) => {
-        expect(err).to.equal(null);
-        expect(res.statusCode).to.equal(200);
-
-        // Expect correct challenge response and signature
-        expect(res.body.response).to.equal('testchallenge');
-        expect(res.headers['x-outbound-token']).to.equal(
-          // Signature of the test body with the Webhook secret
-          '876d1f9de1b36514d30bcf48d8c4731a69500730854a964e31764159d75b88f1');
-
-        check();
+    echo.webapp('testappid', 'testsecret', 'testwsecret')
+      .then((app) => {
```

Comment:
I'd suggest to use await here instead of a promise (since this pull request is about using async/await)